src/workers/postBannedRep.ts

```diff
@@ -32,14 +32,18 @@ const worker: Worker = {
           }),
         );
 
-        const ownerProps = {
-          targetId: id,
-          targetType: ReputationType.Post,
-          reason: ReputationReason.PostBanned,
-        };
-
         // Skipping penalizing community picks during the beta phase
-        if (data.post.sourceId !== COMMUNITY_PICKS_SOURCE) {
+        // Skipping penalizing shared post
+        if (
+          data.post.sourceId !== COMMUNITY_PICKS_SOURCE &&
+          type === PostType.Share
```

Comment:
```suggestion
          type !== PostType.Share
```

Reply:
Oh, right! Completely missed that! Thank you!